virtcontainers/interfaces.go

```diff
@@ -34,6 +34,7 @@ type VC interface {
 	KillContainer(sandboxID, containerID string, signal syscall.Signal, all bool) error
 	StartContainer(sandboxID, containerID string) (VCContainer, error)
 	StatusContainer(sandboxID, containerID string) (ContainerStatus, error)
+	StatsContainer(sandboxID, containerID string) (ContainerStats, error)
```

Comment:
Can you please make this part of the VCSandbox interface and let VC.StatsContainer be a wrapper of it, like we did for the other sandbox APIs?

Reply:
@bergwolf  Got it~ On the way to fix it